### DISCUSSION TITLE
Where are the versions with syntax-highlighting?

### DISCUSSION THREAD

**Discussion by hiensumi** [cpeditor/cpeditor#1302] [Q&A] [answered]
Syntax-highlighting provides lots of themes then why all the versions switched to QCodeEditor instead?

**Comment by hiensumi** [comment]
![image](https://github.com/cpeditor/cpeditor/assets/47080203/831e47f7-5772-4690-977d-438df36f56db)
The ksyntax-highlighting tree of cpeditor is not full :D

**Reply by ouuan** [reply]
You need to install KSH. Either via system package manager, or compiling the submodule.

**Comment by ouuan** [comment, marked as answer]
> why all the versions switched to QCodeEditor instead

We switched from QCodeEditor to KSH, not the opposite.

**Reply by hiensumi** [reply]
Oh I downloaded the full source from the releases which do not have the KSH :)

**Reply by hiensumi** [reply]
No issues here. Thank you!
